oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -319,7 +340,7 @@ Helpful rST-related tools
 
 * Other options are listed on stackoverflow_.
 
-.. _Restructured Text (RST): http://docutils.sourceforge.net/rst.html
+.. _reStructured Text (RST): http://docutils.sourceforge.net/rst.html
```

Comment:
1. `reStructuredText` is one word, as mentioned above.
2. While we are on the topic of Markdown, can we update the bullet point just above this (not yet in the PR) to:
```
* Pandoc_ can be used to convert between many formats, like Google Doc or Markdown to rST.
```